src/components/AccessControlNew.tsx

```diff
@@ -102,9 +103,11 @@ const AccessControlNew = () => {
         setEditName(false)
         dispatch(ruleActions.setRule({
             Name: '',
+            Description: '',
             Source: [],
             Destination: [],
-            Flow: 'bidirect'
+            Flow: 'bidirect',
```

Comment:
Codacy has a fix for the issue: [Replace `····Flow:·'bidirect'` with `Flow:·"bidirect"`](https://app.codacy.com/gh/netbirdio/dashboard/pullRequest?prid=9560373)
```suggestion
        Flow: "bidirect",
```